packages/ember-glimmer/lib/environment.js

```diff
@@ -133,6 +137,18 @@ export default class Environment extends GlimmerEnvironment {
           wrapClassAttribute(args);
           return new CurlyComponentSyntax({ args, definition, templates });
         }
+      } else {
+        // Check if it's a keyword
+        let mappedKey = keywords[key];
+        if (mappedKey) {
+          if (mappedKey !== key) {
+            path = path.map((segment) => segment === key ? mappedKey : segment);
```

Comment:
Is this going to from `textarea` to `-textarea`? Is this because of the `-` requirement for components? If so we should extract this logic out to a `toKeyword` function that does the conversion and does the arg wrapping.